### ISSUE TITLE
A tutorial

### ISSUE THREAD

**Issue by cinaryilmaz** [scratchnative/scratchnative#1]
I have no idea what to do, I tried something and it wasn't successful:
![resim](https://user-images.githubusercontent.com/79412062/174693481-fec88c16-5937-4182-959d-6199a3826155.png)

Please provide a tutorial about making simple applications using this.

**Comment by Tolinme** [comment]
+

**Comment by rdmsr** [comment]
Good idea! What about prefixing external functions with extern though?

**Comment by rdmsr** [comment]
> Good idea! What about prefixing external functions with extern though? 

Like `define procedure extern printf`
